net/seafile-seahub/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=seafile-seahub
 PKG_VERSION:=6.3.4
-PKG_RELEASE:=5
+PKG_RELEASE:=6
 PKG_LICENSE:=Apache-2.0
 
```

Comment:
Add please PKG_LICENSE_FILES, PKG_MAINTAINER.

Reply:
I thought I was maintainer here;
hmm; weird; well, it's Monday and the coffee is just kicking in